setup/web_pwa_oca/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
I don't understand why this file is added in the migration commit, while it should be already present in v12: https://github.com/OCA/web/tree/12.0/setup/web_pwa_oca

Reply:
Maybe this folder is not migrated?

I don't know either, @pedrobaeza can you tell us if I've done something wrong?

Reply:
That's because the procedure to preserve commit history only takes the module directory, not others.

Reply:
@pedrobaeza right, thanks